### DISCUSSION TITLE
Can you make short (<1 minute) clips from longer videos using this?

### DISCUSSION THREAD

**Discussion by Wyyvern** [kmcclive/tricycle#106] [Q&A]
(Reposting this as a new discussion just in case you didn't get a reply notification)

Hi, looking at the screenshots of this it looks pretty good, I'm looking to convert clips from a 4K HDR source down to SDR which Handbrake can't help me with. Just wondering though, is there an option to only convert snippets rather than the whole thing? Kind of like with Handbrake, where you can convert small sections by entering 'from' and 'to' values with timestamps or frame numbers. Thanks ^^

**Comment by kmcclive** [comment]
Hi, @Wyyvern.  Currently, the answer is no.  I created issue #107 to possibly support this in the future.
